### PR TITLE
[12.0] MIG contract_invoice_start_end_dates

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: contract_invoice_start_end_dates/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/contract-10-0/contract-10-0-contract_invoice_start_end_dates
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/110/10.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module automatically add start and end dates to the invoice line
```
Comment (rvalyi) [inline contract_invoice_start_end_dates/README.rst:28]:
```suggestion
This module automatically adds start and end dates to the invoice line
```

**Review by rvalyi — CHANGES_REQUESTED** [review]

File: contract_invoice_start_end_dates/README.rst (hunk 1 above)
Comment (florian-dacosta) [reply contract_invoice_start_end_dates/README.rst:28]: But it seems ok in the splitted readme...
Won't it be automatically re-generated after merge?

**Review by florian-dacosta — APPROVED** [review]
LGTM

File: contract_invoice_start_end_dates/README.rst (hunk 1 above)
Comment (alexis-via) [reply contract_invoice_start_end_dates/README.rst:28]: Yes, it will be regenerated after merge. The rst in the readme subdir is OK.

**Review by rvalyi — APPROVED** [review]

**Comment by alexis-via** [comment]
Travis error is unrelated. So it's ready to merge.

**Comment by alexis-via** [comment]
Conflict solved.

**Pull request merged**